src/sentry/api/endpoints/organization_relay_history.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import absolute_import
+from rest_framework.response import Response
+
+from sentry.api.bases import OrganizationEndpoint, OrganizationPermission
+from sentry.api.serializers import serialize
+from sentry.models import OrganizationOption, RelayUsage
+from sentry import features
+
+
+class OrganizationRelayHistory(OrganizationEndpoint):
+    permission_classes = (OrganizationPermission,)
+
+    def get(self, request, organization):
+        has_relays = features.has("organizations:relay", organization, actor=request.user)
+        if not has_relays:
+            return Response(status=403)
+
+        option_key = "sentry:trusted-relays"
+        try:
+            trusted_relays = OrganizationOption.objects.get(
```

Comment:
_nit_: Note that there is also `OrganizationOption.objects.get_value` or simply `organization.get_option`.